packages/io/stl-deserializer/package.json

```diff
@@ -28,15 +30,23 @@
     "openjscad",
     "jscad",
     "csg",
+    "import",
     "deserializer",
     "stl"
   ],
   "license": "MIT",
   "dependencies": {
-    "@jscad/csg": "0.7.0",
-    "@jscad/io-utils": "^1.0.0-alpha.d0fb3056"
+    "@jscad/modeling": "0.0.1",
+    "@jscad/io-utils": "1.0.0-alpha.d0fb3056"
   },
   "devDependencies": {
-    "ava": "^0.25.0"
+    "ava": "^0.25.0",
+    "eslint": "^5.13.0",
```

Comment:
do we need to have these fore every single dependency ? I think some of the common deps can be moved to a higher level in Lerna ? (can be tackled later, is not essential)